Fix:
error: implicit declaration of function 'close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'hex' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'isatty' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'read' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'write' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- bookinfo.c.orig	2022-01-11 20:16:31.000000000 -0600
+++ bookinfo.c	2022-01-11 20:18:06.000000000 -0600
@@ -42,6 +42,7 @@
 uchr	*dsplist();
 uchr	*dspstyle();
 uchr	*idxinfo();
+int	hex();
 
 int
 main(ac, av)
--- catdump.c.orig	2022-01-11 20:13:33.000000000 -0600
+++ catdump.c	2022-01-11 20:15:21.000000000 -0600
@@ -17,6 +17,7 @@
 #include <fcntl.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <unistd.h>
 
 typedef	unsigned char	uchr;
 
--- squeeze.c.orig	2000-04-13 02:45:30.000000000 -0500
+++ squeeze.c	2022-01-11 13:44:41.000000000 -0600
@@ -17,6 +17,7 @@
 #include <time.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <unistd.h>
 #include "epw.h"
 
 /*
